ghcloneall.py

```diff
@@ -776,6 +782,9 @@ def _main():
     parser.add_argument(
         '--user',
         help='specify the GitHub user')
+    parser.add_argument(
+        '--github-token',
+        help='specify the GitHub token')
```

Comment:
It also would be nice if ghcloneall could load the token from the config file (this needs like three additional lines of code a page or two down from here).

For completeness maybe `ghcloneall --init --github-token XYZZY` should also _write_ the token to the newly created config file (another two lines of code a bit further down).